### PR TITLE
Update Docs when referencing Images

### PR DESCRIPTION
**Pull request by allcentury**
Add manifest instructions and specificy example when using getURL()

### CODE REVIEW

**Comment by allcentury** [comment]
actually, just now seeing this is copy + pasted in other areas, going to move this to a shared doc

**Comment by allcentury** [comment]
@jacksteamdev wdyt?

File: packages/vite-plugin-docs/docs/common/ImageCodeBlock.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import CodeBlock from '@theme/CodeBlock';
+
+// Helper function to dedent the code
+const dedent = (str) => {
+  const lines = str.split('\n');
+  const minIndent = Math.min(
+    ...lines.filter(line => line.trim()).map(line => line.match(/^ */)[0].length)
+  );
+  return lines.map(line => line.slice(minIndent)).join('\n');
```
Comment (jacksteamdev) [inline packages/vite-plugin-docs/docs/common/ImageCodeBlock.js:10]:
```suggestion
  return lines.map(line => line.slice(minIndent)).join('\n').trim();
```
Should we trim the returned code? Looks like there's an extra newline on the top.

image

**Review by jacksteamdev — APPROVED** [review]
@allcentury Great improvement! Let's ship it.

**Pull request merged**